### PR TITLE
32 remove v2 from endpoints

### LINKED ISSUE #32: Remove V2 from endpoints
- Remove /v2 from backend endpoints and have all endpoints exist on the top level

### PR DESCRIPTION
**Pull request by claireolmstead**
# Purpose
The goal of this PR is to remove the `v1` and `v2` prefixes from endpoints.

Closes #32 

## Solution

Remove all `v1/` and `v2/` prefixes from endpoint naming and calls to those endpoints.

### Change summary
* Removed `v1/` and `v2/` prefixes

## Steps to Verify
1. Run swagger and check to see that no endpoints have `v1/` or `v2/` naming
2. When testing an endpoint, you should not see `v1/` or `v2/` in the Request URL.

## Additional details / screenshot

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
```
Comment (mattheworris) [inline .idea/.gitignore:4]: nit: aren't these files ignored by `.idea/`? 🤔

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```
Comment (JoeCap08055) [inline .idea/.gitignore:1]: ➕ to @mattheworris 's comment. Let's not commit the `.idea/` directory

File: backend/controllers/WebhookController.ts [hunk 3]
```diff
@@ -2,16 +2,16 @@ import { Express, Request, Response } from "express";
 import { BaseController } from "./BaseController";
 import { HttpStatusCode } from "axios";
 
-export class  WebhookController extends BaseController {
-    constructor(app: Express) {
-        super(app, '/webhooks');
-    }
+export class WebhookController extends BaseController {
+  constructor(app: Express) {
+    super(app, "/webhooks");
```
Comment (mattheworris) [inline backend/controllers/WebhookController.ts:7]: nit: this repo needs a prettier rule for `single-quote` or `double-quote`, currently it's not consistent.

Comment (JoeCap08055) [reply backend/controllers/WebhookController.ts:7]: `single-quote` seems to be our standard in other repos

**Review by JoeCap08055 — APPROVED** [review]
I'll fix the new test errors in my next PR, since I caused them 😬

**Pull request merged**